metadata/nz_building_outlines_full_history.xml

```diff
@@ -0,0 +1,351 @@
+
+**Source imagery**
+Resolution of the imagery used for capture ranges from 50cm pixel resolution (0.5m GSD) to 7.5cm pixel resolution (0.075m GSD).  Please refer to the [NZ Building Outlines Data Dictionary](https://nz-building-outlines.readthedocs.io/en/latest/introduction.html#source-imagery) for further information.
+
+**Accuracy Specification:**
+The Building Outlines contain spatial detail reflective of the visible characteristics of building outlines as seen from the source imagery. The building outlines were created by an automated method of feature extraction. 
+
+**Guidelines for Building Outline Extraction:**
+**The following rules were used for capturing buildings;**
+• Buildings under construction are not captured.
+Standalone dwellings:
+• Primary building structures are captured as separate polygons. Adjoining buildings are captured as a single polygon.
+• Extensions, sunrooms, balconies, patios, and annexes are captured as part of the primary building structure.
+• Permanent buildings structures such as sheds and greenhouses, not attached to the primary building structure, but larger than 10m² are captured as individual buildings.
+• Inner courtyards within buildings are captured as a hole within the building polygon if resolution of imagery allows. These holes should be no smaller than 10m².
+Townhouses:
+• Adjoining townhouses are captured a single polygon.
+Commercial:
+• Adjoining commercial buildings are captured as separate polygons with coincident shared edges.
+Rural/farmland areas:
+• Hay bales and silage are not captured.
+• Water tanks are captured as buildings. Minimum size is 16.5m².
+General:
+• Non-permanent objects such as shipping containers, caravans, large vehicles and trampolines should not be captured as buildings.
+
+**Spatial Accuracy:**
+Buildings must be;
+• Spatially accurate to 1 metre
+• Captured in full where they are partially occluded by vegetation or require additional viewer interpretation.
+• Squared off (90 degrees) if the corner angles are between 80-100 degrees.      
+
+**Valid Geometry:**    
+A building outline polygon is considered to have valid geometry if;
+• It does not overlap with other building outline polygons;
+• It does not contain any spikes;
+• It does not contain lines that are intersected with each other;
+• It can contain polygons with interior rings (holes);
+• It does not contain polygons with multiple exterior rings;
+• Building polygons captured will be greater than 10m².
+
+Please refer to the [Building Outlines Data Dictionary](https://nz-building-outlines.readthedocs.io/en/latest/published_data.html#table-nz-building-outlines-full-history) for detailed metadata and information about this layer.</gco:CharacterString>
+      </gmd:abstract>
+      <gmd:purpose>
+        <gco:CharacterString>This layer provides the building outlines for mainland New Zealand.</gco:CharacterString>
```

Comment:
`provides the full history of building outlines for..`

Reply:
Done!